js/index.js

```diff
@@ -40,6 +40,26 @@
         });
     }
 
+    window.addEventListener('keydown', function(e) {
```

Comment:
Ok, the main issue I was facing with FontSizeView is that the keydown event was captured only in the input fields and not everywhere. I'll drop this commit and I'll try to fix the FontSizeView.